interface/src/app/tribe_client/directives.js

```diff
@@ -77,4 +77,19 @@ angular.module('adage.tribe_client.directives', [
     };
 }])
 
+// Directive for whole gene search form
+.directive('genesetSearchForm', function( ) {
+    return {
+        controller: ['$scope', function($scope) {
```

Comment:
Yes, for now, but I will reinsert it in the next pull request, when the geneset search is enabled.